ppr-ui/src/composables/userAccess/useUserAccess.ts

```diff
@@ -81,13 +181,52 @@ export const useUserAccess = () => {
     for (const flag in getMhrUserAccessValidation.value) {
       setMhrQsValidation({ key: flag, value: false })
     }
+
+    // Set unsaved changes to prompt cancel dialogs on exit
+    setUnsavedChanges(true)
+  }
+
+  /**
+   * Submit qualified supplier application
+   * Includes a request to CREATE a Qualified Supplier in MHR
+   * Includes a request to CREATE a TASK for Staff in Auth Web
+   */
+  const submitQsApplication = async (): Promise<void> => {
+    const payload: MhrQsPayloadIF = {
+      ...cleanEmpty(getMhrQsInformation.value),
+      authorizationName: getMhrQsAuthorization.value.legalName,
```

Comment:
I think we should update legalName to authorizationName to match the api spec. I set it as legalName before I knew that api was expecting authorizationName.

Reply:
Good call, will update!

Reply:
Updated

Reply:
Un-resolving this comment, while i fix in future commit.